packager/mpd/base/period.cc

```diff
@@ -269,6 +269,20 @@ bool Period::SetNewAdaptationSetAttributes(
       }
     }
 
+    // Set transfer characteristics.
+    // https://dashif.org/docs/DASH-IF-IOP-v4.3.pdf - 4.2.5.1
+    // ISO/IEC 23001-8 MPEG systems technologies — Part 8: Coding-independent
+    // code points. https://en.wikipedia.org/wiki/Coding-independent_code_points
+    // - Common CCIP values.
+    // Dolby vision:
+    // https://professionalsupport.dolby.com/s/article/How-to-signal-Dolby-Vision-in-MPEG-DASH
+    if (new_adaptation_set->codec().find("dvh") == 0) {
+      new_adaptation_set->set_transfer_characteristics(16);
```

Comment:
Please define a reasonably-named constant for this instead of hard-coding 16.  It will be more readable with a name.  What does 16 represent?  The constant's name should answer that question.

Reply:
Updated.